<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebPVClass</class>
 <widget class="QWidget" name="WebPVClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebPV</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_net">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>URLï¼š</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>581</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="webpv.qrc"/>
 </resources>
 <connections/>
</ui>
